<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArrayViewer</class>
 <widget class="QMainWindow" name="ArrayViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>152</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ArrayViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="typeBox">
        <item>
         <property name="text">
          <string>int</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>double</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MyString</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>std::string</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="inputField"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="sortAscButton">
        <property name="text">
         <string>Sort ascending</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="minElemButton">
        <property name="text">
         <string>Min element</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortDescButton">
        <property name="text">
         <string>Sort descending</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="outputField"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
